synapse/state/__init__.py

```diff
@@ -137,22 +137,27 @@ def __init__(self, hs: "HomeServer"):
             ReplicationUpdateCurrentStateRestServlet.make_client(hs)
         )
 
-    async def get_current_state_ids(
+    async def compute_state_after_events(
         self,
         room_id: str,
-        latest_event_ids: Collection[str],
+        event_ids: Collection[str],
     ) -> StateMap[str]:
-        """Get the current state, or the state at a set of events, for a room
+        """Fetch the state after each of the given event IDs. Resolve them and return.
+
+        This is typically used where `event_ids` is a collection of forward extremities
+        in a room, intended to become the `prev_events` of a new event E. If so, the
+        return value of this function represents the state before E.
 
         Args:
-            room_id:
-            latest_event_ids: The forward extremities to resolve.
+            room_id: the room_id containing the given events.
+            event_ids: the events whose state should be fetched and resolved.
 
         Returns:
-            the state dict, mapping from (event_type, state_key) -> event_id
+            the resolution of the states after the given event IDs. This is a mapping
```

Comment:
I have found the term "state dict" useful in the past-would it be amenable to keep it in along with your change (which is edifying), ie something like "the state dict which is a  resolution..[the rest of your addition here]"

Reply:
How do you find [`6f6140e` (#13270)](https://github.com/matrix-org/synapse/pull/13270/commits/6f6140e7f5f9217a945037cc87fa571e10bbcbf4)?